lib/internal/perf/performance.js

```diff
@@ -152,6 +166,11 @@ ObjectDefineProperties(Performance.prototype, {
     enumerable: false,
     value: nodeTiming,
   },
+  markResourceTiming: {
```

Comment:
In the browser, this function is not public. However, it must be used inside `fetch` which is a Node.js deps, not an internal module. Any idea?

Reply:
Once we decide to really export this function publicly, I'll work on the documentation for that.

Reply:
Add a comment?

Reply:
If this isn't part of the spec, could we possibly expose it separately somehow? I'm a tiny bit wary of having non-standard bits on standard interfaces.